src/dead-file.ts

```diff
@@ -35,16 +36,16 @@ async function readSourceFiles(root: string, includeFiles: string[] = [], exclud
   let result: string[] = [];
   const level1Sources = await fs.readdir(root);
   const readAll = level1Sources.filter(isLegalSource).map(async (fileName) => {
-    const subFileName = `${root}/${fileName}`;
-    if (includeFiles.length > 0 && !includeFiles.includes(subFileName)) {
+    const subFileName = resolve(root, fileName);
+    if (includeFiles.length > 0 && !includeFiles.some(includeFile => subFileName.startsWith(includeFile))) {
```

Comment:
"include" and "exclude" at line 40 and line 43 could be better handled with a more sophisticated solution, just got an idea. You could remove these lines.

Reply:
I mean I have a better idea for the "include" & "exclude" config options. To support more syntaxes (regexp, wild card, globbing). So these changes here are not needed.

Reply:
Oh I get it.
But if I revert this, the plugin does not work at all and return `All source files: 0` (which of course is not true :D )
I recommend to accept the fix for now to have a working release, then improve with your solution.